lib/archethic/replication.ex

```diff
@@ -564,14 +565,15 @@ defmodule Archethic.Replication do
   - Transactions with smart contract deploy instances of them or can put in pending state waiting approval signatures
   - Code approval transactions may trigger the TestNets deployments or hot-reloads
   """
-  @spec ingest_transaction(Transaction.t(), boolean()) :: :ok
-  def ingest_transaction(tx = %Transaction{}, io_transaction?) do
+  @spec ingest_transaction(Transaction.t(), boolean(), boolean()) :: :ok
+  def ingest_transaction(tx = %Transaction{}, io_transaction?, self_repair? \\ false) do
+    # also called from bootstrap
```

Comment:
I don't get this commentary, this function is not called from bootstrap. Only in replication and bootstrap_test